apiserver/restrict_caasmodel.go

```diff
@@ -13,8 +13,10 @@ import (
 // commonModelFacadeNames lists facades that are shared between CAAS
 // and IAAS models.
 var commonModelFacadeNames = set.NewStrings(
+	"ActionPruner",
```

Comment:
why did you add ActionPruner, CharmRevisionUpdater, and StatusHistory? are they relevant to this PR? if not, please add later when they are required

Reply:
CAAS models start the relevant workers

Reply:
I see, they were just missing from before.